aiohttp/worker.py

```diff
@@ -159,6 +161,11 @@ def _create_ssl_context(cfg):
             ctx.set_ciphers(cfg.ciphers)
         return ctx
 
+    def _fix_log_format(self, source_format, default_format):
+        if re.search(r'%\([^\)]+\)', source_format):
+            return default_format
```

Comment:
Seems this is done. Also added a not to docs "Logging" section.

![image](https://cloud.githubusercontent.com/assets/957767/17963104/97eca0f8-6abc-11e6-95df-6a8add564921.png)